standalone/instructlab-pipelines-with-dsp-on-rhoai-.md

```diff
@@ -0,0 +1,646 @@
+> are then provided as a parameter to the standalone.py script as well as in the `judge-serving-details` secret above.
+
+If you deployed the Judge server model using the optional instructions above then you can retrieve `JUDGE_API_KEY` by
+running the following command:
+
+```bash
+JUDGE_API_KEY=$(oc -n <data-science-project-name/namespace> create token judge-sa)
+```
+
+#### Deploy a teacher model server (Optional)
+
+Unlike the Judge model we have to deploy the Teacher model manually on RHOAI, this consists of deploying the K8s resources
+using `oc`.
+
+First, upload the Teacher model to s3 if it does not already exist there:
+
+```bash
+# You can also use Oras or Skopeo cli tools to download the model
+# If using other tools besides ilab, ensure that filenames are mapped
+# appropriately
+ilab model download --repository docker://registry.redhat.io/rhelai1/mixtral-8x7b-instruct-v0-1 --release 1.2
+
+# Default cache location for ilab model download is ~/.cache/instructlab/models
+# The model should be copied in such a way that the *.safetensors are found in s3://your-bucket-name/teach-model/*.safetensors
+s3cmd sync path/to/model s3://your-bucket-name/teach-model/
+```
+
+Deploy the following `yaml` called `pre_requisites.yaml` to the `<data-science-project-name/namespace>`
+<details>
+
+<summary>pre_requisites.yaml</summary>
+
+```yaml
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: mixtral-sa
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: mixtral-view-role
+  labels:
+    opendatahub.io/dashboard: 'true'
+rules:
+  - verbs:
+      - get
+    apiGroups:
+      - serving.kserve.io
+    resources:
+      - inferenceservices
+    resourceNames:
+      - mixtral
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: mixtral-view
+  labels:
+    opendatahub.io/dashboard: 'true'
+subjects:
+  - kind: ServiceAccount
+    name: mixtral-sa
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: mixtral-view-role
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: mixtral-serving-ilab
+  labels:
+    opendatahub.io/dashboard: 'true'
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 200Gi
+  storageClassName: standard-csi
+  volumeMode: Filesystem
+```
+
+</details>
+
+```bash
+oc -n <data-science-project-name/namespace> apply -f pre_requisites.yaml
+```
+
+You will need to ensure that the `storage-config` secret exists in the `<data-science-project-name/namespace>` namespace.
+And this `storage-config` has the configuration for the bucket where the teacher model is stored.
+
+```yaml
+apiVersion: v1
+stringData:
+  aws-connection-my-bucket: |
+    {
+      "type": "s3",
+      "access_key_id": "your_accesskey",
+      "secret_access_key": "your_secretkey",
+      "endpoint_url": "https://s3-us-east.amazonaws.com",
```

Comment:
minor nitpick: the region is `us-east-1` or `us-east-2`.  that said, perhaps we should suggest something generic since we're also providing `mybucket`, `your_accesskey`, etc